src/ImageSharp.Web/Commands/Converters/ArrayConverter{T}.cs

```diff
@@ -18,28 +17,30 @@ public sealed class ArrayConverter<T> : ICommandConverter<T[]>
 
     /// <inheritdoc/>
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
-    public T[] ConvertFrom(
+    public bool TryConvertFrom(
         CommandParser parser,
         CultureInfo culture,
-        string value,
-        Type propertyType)
+        string? value,
+        Type propertyType,
+        out T[] convertedValue)
     {
         if (string.IsNullOrWhiteSpace(value))
         {
-            return Array.Empty<T>();
+            convertedValue = Array.Empty<T>();
```

Comment:
Ah.... OK. So in some cases we do and some we don't. 

This should be more consistant and return false on empty input.

Reply:
yeah and in some cases we convert to an struct so there is no null..

Reply:
I'm gonna pull this down and have a play.